sct-commons/src/main/java/org/lfenergy/compas/sct/commons/DoTypeService.java

```diff
@@ -25,4 +32,59 @@ public Optional<TDOType> findDoType(TDataTypeTemplates tDataTypeTemplates, Predi
         return getFilteredDoTypes(tDataTypeTemplates, tdoTypePredicate).findFirst();
```

Comment:
Same comment for `findDoType` as `findLnodeType`

Reply:
This method I didn't create, I just use it.